File: RUST/Cap05/c05ex11.rs
// c05ex11.rs

use std::io::{self, Write};

fn main() {
  for _i in 1..=5 { // ou use: for _i in 1..=5.step_by(1) {
    print!("Entre um valor numerico: ");
    io::stdout().flush().expect("Falha ao limpar o buffer");
    let mut entra_n = String::new();
    io::stdin().read_line(&mut entra_n).expect("Falha ao ler a entrada");
    let n: i32 = entra_n.trim().parse().expect("Entrada inválida");

    let r = n * 3;

    println!("Resultado = {}", r);
    println!();
  }

  println!();
  print!("Tecle <Enter> para encerrar... ");
  io::stdout().flush().expect("Falha ao limpar o buffer");
  let mut _entrada = String::new();
  io::stdin().read_line(&mut _entrada).expect("Falha ao ler a entrada");
}

File: CSHARP/Cap05/c05ex11.cs
// c05ex11.cs

using System;
using static System.Console;

class Program
{
    static void Main(string[] args)
    {
        int I, N, R;

        for (I = 1; I <= 5; I++)
        {
            Write("Entre um valor numerico: ");
            N = int.Parse(ReadLine());

            R = N * 3;

            WriteLine($"Resultado = {R}\n");
        }

        Write("\nTecle <Enter> para encerrar... ");
        ReadLine();
    }
}
